infra-l2-dynamo/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "dca-mock-readid",
+    "description": "Mock READID endpoints for use in e2e tests",
+    "version": "0.0.1",
+    "private": true,
+    "dependencies": {
+        "esbuild": "^0.15.16"
+    },
+    "devDependencies": {
+        "@types/aws-lambda": "^8.10.92",
+        "@types/jest": "^27.5.0",
+        "@types/node": "^17.0.13",
+        "@aws-cdk/assertions": "^1.172.0",
+        "@types/js-yaml": "^4.0.5",
+        "esbuild-jest": "^0.5.0",
+        "jest": "^29.0.0",
+        "jest-junit": "^14.0.1",
+        "ts-node": "^10.4.0",
+        "typescript": "^4.7.3",
+        "yaml-cfn": "^0.3.1"
+    },
+    "scripts": {
+        "test": "./node_modules/.bin/jest",
```

Comment:
needed?

Reply:
Yeah, this will be being used to execute the infra tests.

Reply:
```suggestion
        "test": "npm run infra",
```